src/test/javascript/spec/service/programming-exercise.service.spec.ts

```diff
@@ -162,6 +168,15 @@ describe('ProgrammingExercise Service', () => {
             const req = httpMock.expectOne({ method: 'DELETE' });
             req.flush({ status: 200 });
         });
+
+        it('should make get request', fakeAsync(() => {
+            const expect = chai.expect;
+            const expectedBlob = new Blob(['abc', 'cfe']);
+            service.exportInstructorExercise(123).subscribe((resp) => expect(resp.body).equal(expectedBlob));
+            const req = httpMock.expectOne({ method: 'GET', url: `${resourceUrl}/123/export-instructor-exercise` });
+            req.flush(expectedBlob);
+            tick();
+        }));
```

Comment:
@iozbeyli I tried to refactor the test I had to add as described #3305 , as in this file chai and sinon were not used before, I had to do it this to not mess up the `expect` statements in the other test cases. And furthermore, this is now a file were different testing libraries are mixed up, I guess. Do you think it is still reasonable to do it like this and cover a refactoring of tests this file in another PR?

Reply:
You can use toEqual from Jest instead of equal from chai. I think #3305 also has toEqual instead of equal here.